src/Pickles/Pickles/FeatureParser.cs

```diff
@@ -90,5 +92,18 @@ private string DetermineLanguage()
             }
             return language;
         }
+
+        private Feature RemoveFeatureWithExcludeTags(Feature result)
+        {
+            if (result.Tags.Any(t => t.Equals($"@{configuration.ExcludeTags}", StringComparison.InvariantCultureIgnoreCase)))
+                return null;
+
+            var wantedFeatures = result.FeatureElements.Where(fe => fe.Tags.All(t => !t.Equals($"@{configuration.ExcludeTags}", StringComparison.InvariantCultureIgnoreCase))).ToList();
```

Comment:
In Then_can_parse_and_ignore_scenario_with_tag_in_configuration_ignore_tag_and_keep_feature
` @scenario-tag-1 @scenario-tag-2 @Ignore-Tag`  and `@scenario-tag-1 @scenario-tag-2 @ignore-tag`

But if you have missed it, it isn't enough clear and a test is needed